chain/gen/genesis/genesis.go

```diff
@@ -580,6 +580,23 @@ func MakeGenesisBlock(ctx context.Context, j journal.Journal, bs bstore.Blocksto
 		return nil, xerrors.Errorf("setup miners failed: %w", err)
 	}
 
+	if template.NetworkVersion >= network.Version16 {
```

Comment:
Might not be easy, but we should make the genesis logic use the FVM. Definitely not needed now, but this is the only thing which depends on a big chunk of Go specs-actors logic

Reply:
(definitely not something we want to do in this PR / not needed for M1)

Reply:
yeah, agreed. The sooner specs-actors retires, the better.

Reply:
Oki